docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -391,7 +425,53 @@ definitions:
     items:
       $ref: '#/definitions/Group'
   Group:
-    $ref: './schemas/group-schema.json'
+    type: object
```

Comment:
If we use yaml is it still possible to extract the schemas for data types into a separate file, in order to keep `hypothesis.yaml` to a size where it is still easy to skim through?

Reply:
Absolutely something on my list to figure out—this PR is not comprehensive (to say the least) in terms of moving toward an ideal organization of stuff. I agree that the current YAML file could get unwieldy.